src/libs/Navigation/linkingConfig/index.ts

```diff
@@ -20,4 +20,5 @@ const linkingConfig: LinkingOptions<RootStackParamList> = {
     config,
 };
 
-export default linkingConfig;
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I see that we have started to ignore this rule in many places. Is it still relevant, then?

Reply:
Yeah Fabio asked me same question but there are places where its still relevant I think